src/sentry/models/organization.py

```diff
@@ -138,9 +138,6 @@ class Organization(Model):
             ), (
                 'require_2fa',
                 'Require and enforce two-factor authentication for all members.'
-            ), (
-                'disable_new_visibility_features',
-                'Temporarily opt out of new visibility features and ui',
```

Comment:
Should we bulk update the flags for organizations? Next time when we add a flag some users will have state in that bit.

Reply:
@markstory Good call, I think we should tackle this in Sentry 10 cleanups. I added back these lines here so we don't forget about it later.